components/console-backend-service/README.md

```diff
@@ -80,7 +80,7 @@ dep ensure -vendor-only
 This project uses the [GQLGen](https://github.com/99designs/gqlgen) library, which improves development by generating code from the [GraphQL schema definition](internal/gqlschema/schema.graphql).
 
 1. Define types and their fields in `/internal/gqlschema/schema.graphql` using the [Schema Definition Language](http://graphql.org/learn/schema/).
-1. Execute the `./gqlgen.sh` script to run the code generator.
+1. Execute the `go generate ./...` to run the code generator.
```

Comment:
```suggestion
1. Run the code generator with the `go generate ./...` command.
```

Also, where do you need to execute the command from? 
It would be advisable to add that information as a preceding point in the list if it's another directory, sth like: 
```
1. Navigate to `X`.
```
or in the same point if it's the project directory
```
1. From the project directory, run the code generator with the `go generate ./...` command.
```